apps/vercel/app-actions/src/actions/get-preview-envs.spec.ts

```diff
@@ -36,6 +21,7 @@ describe('get-preview-env.handler', () => {
 
   it('returns a list of preview environments', async () => {
     const result = await handler(callParameters, context);
+    console.log({ result });
```

Comment:
is this temporary or do we want to 🧹 ?

Reply:
Yeah good catch. I will get rid of it in my fast follow PR.